v2/handler.go

```diff
@@ -0,0 +1,110 @@
+package jsonrpc
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"sync"
+)
+
+// DefaultHandler is the default Handler for serving requests quickly
+var DefaultHandler = &Handler{
+	Dispatcher: DefaultDispatcher,
+}
+
+// Handler provides the interface between http requests and the Dispatcher
+type Handler struct {
+	Dispatcher Dispatcher
+}
+
+func serverError(w http.ResponseWriter, message string, code int) {
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
+
+	result := Response{
+		Version: "2.0",
+		ID:      nil,
+		Error: &Error{
+			Code:    code,
+			Message: message,
+		},
+	}
+
+	resp, _ := Marshal(&result)
```

Comment:
Kinda understand you don't want to handle error in serveError fn :D but could be useful to just log it even if you won't return anything